src/main/kotlin/com/sourcegraph/cody/chat/ui/MessagePart.kt

```diff
@@ -29,6 +29,8 @@ class CodeEditorPart(
     val attribution: AttributionButtonController
 ) : MessagePart {
 
+  private var lastIndex = 0
```

Comment:
Do we really need that tmp variable? Cannot we use `editor.document.textLength` instead?

Reply:
This code has been changed. Please take a look.

Reply:
My question remains valid.
I do not see why we cannot use `editor.document.textLength` and `editor.document.text` instead of tmp variables `lastIndex` and `text`.

Reply:
I've removed this code.